.github/merge_rules.json

```
@@ -188,6 +188,90 @@
          "Lint",
          "pull"
       ]
+    },
+    {
+      "name": "IDEEP",
+      "patterns": [
+        "third-party/ideep/**",
```

I though it should be something like the following
```suggestion
        "third_party/ideep",
```